pkg/dockerregistry/server/auth.go

```diff
@@ -375,7 +375,7 @@ func getOpenShiftAPIToken(req *http.Request) (string, error) {
 }
 
 func verifyOpenShiftUser(ctx context.Context, c client.UsersInterfacer) (string, string, error) {
-	userInfo, err := c.Users().Get("~", metav1.GetOptions{})
+	userInfo, err := c.Users().Get(context.TODO(), "~", metav1.GetOptions{})
```

Comment:
Use `ctx` instead of `context.TODO()`.

Reply:
Good catch!

Reply:
I think that this problem happens in a lot of different places, I will fix them and submit a new commit. Please hold on.

Reply:
I sent a new commit that address context's usage, ptal.